catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogCapability.java

```diff
@@ -25,4 +25,18 @@ public CapabilityResult columnDefaultValue() {
         "The DEFAULT constraint for column is only supported since Hive 3.0, "
             + "but the current Gravitino Hive catalog only supports Hive 2.x.");
   }
+
+  @Override
+  public CapabilityResult caseSensitiveOnName(Scope scope) {
```

Comment:
Where did you put the capability restrictions to the catalog scope?

Reply:
I'm afraid the cability of Catalog scope cannot be managed within the framework. For example, when altering a catalog, we need to first obtain the capability of the catalog, which requires the provider and properties of the catalog. However, we only have the `NameIdentifier` of the catalog. So I plan to remove the catalog scope. Do you have a better solution?

Reply:
So how do we restrict the metalake and catalog name?

Reply:
added, please help review again